worksheet.rb

```diff
@@ -31,4 +69,40 @@
 # - the total amount of money each driver has made
 # - the average rating for each driver
 # - Which driver made the most money?
-# - Which driver has the highest average rating?
\ No newline at end of file
+# - Which driver has the highest average rating?
+
+
+def calculate_driver_stats(drivers)
+
+  earnings = {}
+  avg_ratings = {}
+
+  drivers.each do |driver, rides|
+
+    total_earnings = rides.reduce(0) { |sum, ride| sum + ride[:cost] }
```

Comment:
Good job getting `reduce` working, however you can clarify this with `sum`:

```suggestion
    total_earnings = rides.sum { |ride| ride[:cost] }
```